apps/omg_child_chain/test/omg_child_chain/integration/fixtures.exs

```diff
@@ -96,4 +98,15 @@ defmodule OMG.ChildChain.Integration.Fixtures do
 
     {deposit_blknum, token_deposit_blknum}
   end
+
+  defp wait_for_web() do
```

Comment:
Is there an overall timeout? From the look it could leave the test stalling indefinitely?

Same in other modules.

Reply:
added 10seconds